app/components/HeroSection.tsx

```diff
@@ -0,0 +1,18 @@
+import heroImg from "../imgs/Hero-Image.png"
+
+export default function HeroSection() {
+  return (
+    <section className="bg-[#E4EFF0] pt-10 pb-40">
```

Comment:
Is this a color we will use a lot (`bg-[#E4EFF0])`)? If so, we can add it to the `tailwind.config` file and give it a nice name like "primary" or "warm" and use it like this: `bg-warm`